libs/entry-components/configure-theming.md

```diff
@@ -0,0 +1,491 @@
+- `font`: Specifies the default font color.
+- `disabled`: Style elements that are in a disabled state:
+  - `foreground`: Text color for disabled elements
+  - `background`: Background color for disabled elements
+
+```scss
+$custom-theme: (
+  general: (
+    density: 0,
+    colors: (
+      primary: #2581C4,
+      accent: #EA518D,
+      font: #323232,
+      disabled: (
+        foreground: rgb(0 0 0 / .38),
+        background: rgb(0 0 0 / .12)
+      )
+    )
+  )
+);
+```
+
+#### Fonts
+
+Allow customization of typography and font styles based od Angular Material typography levels. Each font related property in configuration can have **family** and **size** values.
+
+- `hero-titles`: Define typography for h1, h2, h3, h4 elements.
+- `titles`: Typography h3, h4 elements
+- `body`: Define typography for base body text.
+- `buttons`: Typography for buttons and anchors
+
+> **_NOTE:_**  Don't forget to check whether font is being applied by checking font families.
+
+```scss
+$custom-theme: (
+  general: (
+    density: 0,
+    colors: (
+      primary: #2581C4,
+      accent: #EA518D,
+      font: #323232,
+      disabled: (
+        foreground: rgb(0 0 0 / .38),
+        background: rgb(0 0 0 / .12)
+      )
+    ),
+    fonts: (
+      hero-titles: (
+        family: 'Helvetica',
+        size: 40px
+      ),
+      titles: (
+        family: 'Roboto',
+        size: 25px
+      ),
+      body: (
+        family: 'Helvetica',
+        size: 20px
+      ),
+      buttons: (
+        family: 'Roboto',
+        size: 10px
+      )
+    ),
+  )
+);
+```
+
+To apply it on whole body of the app, add `.mat-body` class to the body element in index.html file in app root
+
+```html
+<!-- Applying mat-body class to the root component in index.html -->
+<body class="mat-body">
+<app-root></app-root>
+</body>
+```
+
+Since we're overriding Angular Material, it is important to add CSS classes for mixin that emits styles for native header elements scoped within the .mat-typography CSS class
+
+| CSS class      | Example                                       |
+| -------------- | --------------------------------------------- |
+| .mat-headline-1   | `<h1 class="mat-headline-1">{{title}}</h1>`|
+| .mat-headline-2   | `<h2 class="mat-headline-2">{{title}}</h2>`|
+
+#### Spacing
+
+The submap providing the necessary spacing information for styling the **entry form**, **search form button** and **form field**
+
+```scss
+$custom-theme: (
+  general: (
+    spacing: (
+      default: 15px
+    ),
+  )
+);
+```
+
+#### Inputs
+
+We utilize the background value within the inputs configuration to establish the background color for **form text fields**. By default, background property has no color explicitly defined, allowing the default Material styling or any existing CSS rules to take effect.
+
+```scss
+$custom-theme: (
+  general: (
+    spacing: (
+      default: 15px
+    ),
+    inputs: (
+      background: 15px
```

Comment:
Colors? Not pixels?

Reply:
That's my oversight. It was unintentional